pennylane/ops/qubit/state_preparation.py

```diff
@@ -88,6 +86,28 @@ def compute_decomposition(n, wires):
         """
         return [BasisStatePreparation(n, wires)]
 
+    def state_vector(self, wire_order=None):
+        """Returns a state-vector of shape ``(2,) * num_wires``."""
+        state = self.parameters[0]
```

Comment:
Just a minor comment. We should probably use a different term than `state`, since that has a pretty strong association with the `[2] * num_wires` object we are returning.  

Elsewhere in the operation, this value is called *n*. Which still isn't the most communicative name, but it might still be a better variable name than `state`.

Reply:
called it `prep_vals` in the newest commit bc it's an array of integers to prepare the state! (and I think `prep_ints` feels worse, particularly because it makes me think each element can be _any_ int)